galaxy/templates/home_content.html

```diff
@@ -1,33 +1,17 @@
+{% load contentblock %}
+
 <div id="galaxy-info">
     <div class="container">
         <div class="row">
             <div class="col-lg-4 col-md-4 col-sm-4">
-                <div class="download-icon"><i class="fa fa-cloud-download fa-2x"></i></div>
-                <div class="info-label">Download</div>
-                <div class="content-txt">
-                    <p>Jump-start your automation project with great content from the Ansible community. Galaxy provides pre-packaged units of work known to 
-                    Ansible as <a href="http://docs.ansible.com/playbooks_roles.html#roles" target="_blank">roles</a>.</p>
-                    <p>Roles can be dropped into Ansible PlayBooks and immediately put to work. You'll find roles for provisioning infrastructure, deploying 
-                    applications, and all of the tasks you do everyday.</p>
-                    <p>Use <a href="/list#/roles">Search</a> to find roles for your project, then download them onto your Ansible host using the
-                    <a href="http://docs.ansible.com/ansible/galaxy.html#the-ansible-galaxy-command-line-tool" target="_blank">ansible-galaxy</a> command that 
-                    comes bundled with Ansible.</p>
-                    <p>For example:</p>
-<pre>
-$ ansible-galaxy install username.rolename
-</pre>
-               </div>
+                {% contentblock main-downloads %}
```

Comment:
Consider using single tag for rendering block content, e.g. `{% contentblock main-downloads %}`.